### PR TITLE
ci: Add runtime jenkinsfile

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-runtime

**Comment by nebril** [comment]
test-runtime

**Comment by nebril** [comment]
test-runtime

File: jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        stage ("Copy code and boot vms"){
+            options {
+                timeout(time: 30, unit: 'MINUTES')
+            }
+
+            environment {
+            CONTAINER_RUNTIME=setIfLabel("area/containerd", "containerd", "docker")
+                GOPATH="${WORKSPACE}/${TESTED_SUITE}-gopath"
+                TESTDIR="${GOPATH}/${PROJ_PATH}/test"
+                KUBECONFIG="vagrant-kubeconfig"
```
Comment (raybejjani) [inline jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile:102]: The tabbing on these lines is different for some reason

File: vagrant_box_defaults.rb [hunk 2]
```diff
@@ -8,3 +8,5 @@
 $NETNEXT_SERVER_VERSION= "53"
 @v419_SERVER_BOX= "cilium/ubuntu-4-19"
 @v419_SERVER_VERSION= "12"
+@v49_SERVER_BOX="cilium/ubuntu"
```
Comment (raybejjani) [inline vagrant_box_defaults.rb]: Isn't this part of https://github.com/cilium/cilium/pull/11061?

File: jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        stage ("BDD-Test-PR"){
+            options {
+                timeout(time: 180, unit: 'MINUTES')
+            }
+            environment {
+                GOPATH="${WORKSPACE}/${TESTED_SUITE}-gopath"
+                TESTDIR="${GOPATH}/${PROJ_PATH}/test"
+            }
+            steps {
+                sh 'cd ${TESTDIR}; ginkgo --focus="$(python get-gh-comment-info.py "${ghprbCommentBody}" | sed "s/^$/Runtime.*/" | sed "s/K8s.*/NoTests/")" -v --failFast=${FAILFAST} -- -cilium.provision=false -cilium.timeout=${GINKGO_TIMEOUT}'
```
Comment (raybejjani) [inline jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile:135]: Is the python script new?

File: vagrant_box_defaults.rb (hunk 2 above)
Comment (nebril) [reply vagrant_box_defaults.rb]: yeah, will probably need to rebase after it's merged

File: jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile (hunk 3 above)
Comment (nebril) [reply jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile:135]: it's used in kernel-specific jenkinsfile

**Comment by nebril** [comment]
test-runtime

**Comment by nebril** [comment]
retest-runtime

**Review by aanm — APPROVED** [review]

**Pull request merged**